src/topology/metric_space/lipschitz.lean

```diff
@@ -50,6 +50,15 @@ variables [emetric_space α] [emetric_space β] [emetric_space γ] {K : ℝ≥0}
 
 lemma edist_le (h : lipschitz_with K f) (x y : α) : edist (f x) (f y) ≤ K * edist x y := h x y
 
+lemma edist_ge (h : lipschitz_with K f) (x y : α) :
+  (K⁻¹ : ennreal) * edist (f x) (f y) ≤ edist x y :=
```

Comment:
I don't like this pair of names (`edist_le`, `edist_ge`) but failed to come up with another pair of short names.